main.nf

```diff
@@ -351,14 +339,21 @@ include { QC_TRIM } from './modules/nf-core/subworkflow/qc_trim'
 
 workflow {
 
+/*
+================================================================================
+                                  BUILD INDICES
+================================================================================
+*/
+
     BUILD_INDICES(
         dbsnp,
         fasta,
         germline_resource,
         known_indels,
         pon,
         step,
-        tools)
+        tools,
+        modules['bwa_index'])
```

Comment:
Nice, I didn't know modules can be directly referenced like that

Reply:
this is the params.modules actually, I choosed to reassign it as modules, so that I can actually change some configuration in the script